recipes/openexr/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set name = "openexr" %}
+{% set version = "2.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # The release tarball cannot be used from windows. Using the git repository
+  # for the windows build.
+
+  # Windows
+  url: https://github.com/openexr/{{ name }}/archive/v{{ version }}.tar.gz  # [win]
+  sha256: 8243b7de12b52239fe9235a6aeb4e35ead2247833e4fbc41541774b222717933  # [win]
+
+  # Unix
+  url: https://github.com/openexr/openexr/releases/download/v{{ version }}/{{ name|lower }}-{{ version }}.tar.gz  # [not win]
+  sha256: fd6cb3a87f8c1a233be17b94c74799e6241d50fc5efd4df75c7a4b9cf4e25ea6  # [not win]
+
+build:
+  number: 0
+  # skip: True  # [win and vc<14]
+  skip: True  # [win]
+  run_exports:
+    # https://abi-laboratory.pro/?view=timeline&l=openexr
+    - {{ pin_subpackage(name, max_pin="x.x") }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+    - zlib
+    - ilmbase
+  run:
+    - zlib
+    - ilmbase
+
+test:
+  commands:
+    - test -e $PREFIX/include/OpenEXR/OpenEXRConfig.h  # [unix]
+    - test -e $PREFIX/include/OpenEXR/IlmBaseConfig.h  # [unix]
+    - test -e $PREFIX/lib/libHalf$SHLIB_EXT            # [unix]
+    - test -e $PREFIX/lib/libIexMath$SHLIB_EXT         # [unix]
+    - test -e $PREFIX/lib/libIex$SHLIB_EXT             # [unix]
+    - test -e $PREFIX/lib/libIlmImf$SHLIB_EXT          # [unix]
+    - test -e $PREFIX/lib/libIlmImfUtil$SHLIB_EXT      # [unix]
+    - test -e $PREFIX/lib/libIlmThread$SHLIB_EXT       # [unix]
+    - test -e $PREFIX/lib/libImath$SHLIB_EXT           # [unix]
+
+about:
+  home: https://github.com/openexr/openexr
+  license: Modified BSD
```

Comment:
We are trying to standardize this based on the SPDX. Can you change that BSD-3-Clause?

Reply:
Sounds good. Doing it on the feedstock.

Reply:
Fixed.